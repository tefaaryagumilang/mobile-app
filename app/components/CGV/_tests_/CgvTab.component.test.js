import React from 'react';
import renderer from 'react-test-renderer';

import CgvTab from '../CgvTab.component';

describe('CgvTab component', () => {
  xit('CgvTab: renders correctly', () => {
    const tree = renderer.create(<CgvTab />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
