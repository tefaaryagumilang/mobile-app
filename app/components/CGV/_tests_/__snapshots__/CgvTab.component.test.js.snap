// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CgvTab component CgvTab: renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#F0F0F0",
      "flex": 1,
      "justifyContent": "space-between",
      "paddingRight": 0,
    }
  }
>
  <View
    onLayout={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  >
    <View
      onLayout={[Function]}
      style={
        Array [
          Object {
            "borderColor": "#ccc",
            "borderLeftWidth": 0,
            "borderRightWidth": 0,
            "borderTopWidth": 0,
            "borderWidth": 1,
            "height": 50,
          },
          Object {
            "backgroundColor": "#FFFFFF",
          },
          Object {},
        ]
      }
    >
      <RCTScrollView>
        <View>
          <View
            onLayout={[Function]}
            style={
              Array [
                Object {
                  "flexDirection": "row",
                  "justifyContent": "space-around",
                },
                Object {
                  "width": null,
                },
                Object {},
              ]
            }
          >
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel="Sedang tayang"
              accessibilityTraits="button"
              accessible={true}
              hitSlop={undefined}
              isTVSelectable={true}
              onLayout={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
              testID={undefined}
              tvParallaxProperties={undefined}
            >
              <View
                style={
                  Array [
                    Object {
                      "alignItems": "center",
                      "height": 49,
                      "justifyContent": "center",
                      "paddingLeft": 20,
                      "paddingRight": 20,
                    },
                    Object {},
                  ]
                }
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  ellipsizeMode="tail"
                  style={
                    Array [
                      Object {
                        "color": "#000000",
                        "fontWeight": "bold",
                      },
                      undefined,
                    ]
                  }
                >
                  Sedang tayang
                </Text>
              </View>
            </View>
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel="Cinema"
              accessibilityTraits="button"
              accessible={true}
              hitSlop={undefined}
              isTVSelectable={true}
              onLayout={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
              testID={undefined}
              tvParallaxProperties={undefined}
            >
              <View
                style={
                  Array [
                    Object {
                      "alignItems": "center",
                      "height": 49,
                      "justifyContent": "center",
                      "paddingLeft": 20,
                      "paddingRight": 20,
                    },
                    Object {},
                  ]
                }
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  ellipsizeMode="tail"
                  style={
                    Array [
                      Object {
                        "color": "#9E9E9E",
                        "fontWeight": "normal",
                      },
                      undefined,
                    ]
                  }
                >
                  Cinema
                </Text>
              </View>
            </View>
            <View
              accessibilityComponentType={undefined}
              accessibilityLabel="Segera hadir"
              accessibilityTraits="button"
              accessible={true}
              hitSlop={undefined}
              isTVSelectable={true}
              onLayout={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
              testID={undefined}
              tvParallaxProperties={undefined}
            >
              <View
                style={
                  Array [
                    Object {
                      "alignItems": "center",
                      "height": 49,
                      "justifyContent": "center",
                      "paddingLeft": 20,
                      "paddingRight": 20,
                    },
                    Object {},
                  ]
                }
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  ellipsizeMode="tail"
                  style={
                    Array [
                      Object {
                        "color": "#9E9E9E",
                        "fontWeight": "normal",
                      },
                      undefined,
                    ]
                  }
                >
                  Segera hadir
                </Text>
              </View>
            </View>
            <View
              style={
                Object {
                  "backgroundColor": "#ED1D25",
                  "bottom": 0,
                  "height": 4,
                  "left": 0,
                  "position": "absolute",
                  "width": 0,
                }
              }
            />
          </View>
        </View>
      </RCTScrollView>
    </View>
    <RCTScrollView>
      <View>
        <View
          style={
            Object {
              "width": 750,
            }
          }
        >
          <RCTScrollView>
            <View>
              <View />
              <View
                style={
                  Object {
                    "justifyContent": "space-between",
                    "paddingHorizontal": 20,
                    "paddingTop": 20,
                  }
                }
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "color": "#949494",
                      "textAlign": "justify",
                    }
                  }
                >
                  Tiket dijual oleh CGV. Jika terdapat Biaya admin dan diskon, maka diatur oleh Bank Sinarmas
                </Text>
              </View>
            </View>
          </RCTScrollView>
        </View>
        <View
          style={
            Object {
              "width": 750,
            }
          }
        >
          <View
            tabLabel="Cinema"
          />
        </View>
        <View
          style={
            Object {
              "width": 750,
            }
          }
        >
          <View
            tabLabel="Segera hadir"
          />
        </View>
      </View>
    </RCTScrollView>
  </View>
</View>
`;
